site/web/app/themes/sage/assets/styles/components/resource-card.sass

```diff
@@ -27,16 +27,12 @@
         -moz-text-decoration-color: #E52D27
         font-size: 14px
 
-.resources:after
-  content: ""
-  display: table
-  clear: both
+section.resources
+    display: flex
```

Comment:
Good!
So you would use width 100% and padding of whatever to this container. Let's also make a media query for that as well, maybe 45px on the left and right for the styling then under 768px use 15px for the padding left and right